modules/core/src/main/scala/eu/timepit/scalasteward/application/Config.scala

```diff
@@ -45,6 +45,8 @@ import scala.sys.process.Process
   */
 final case class Config(
     workspace: File,
+    reposFile: File,
+    reposDir: File,
```

Comment:
What is the reason for adding `reposDir`? Why would you want to override the default `workspace / "repos"`?

Reply:
yeah, on a second thought I guess it doesn't make that much sense now 😄